Toolset/palettes/script editor/behaviors/revsestackbehavior.livecodescript

```diff
@@ -2144,6 +2144,13 @@ command actionAutocompleteSnippetManager
    modeless "com.livecode.palette.autocomplete-pro"
 end actionAutocompleteSnippetManager
 
+command actionLaunchMessageBox
+   local tStackName
+   put revIDEPaletteToStackName("Message Box") into tStackName
```

Comment:
You don't need this part - revIDEOpenPalette maps to the stack name

Reply:
ok, done